### PR TITLE
README: Do not add all possible workflows

### PR DESCRIPTION
**Pull request by glensc**
|    Q          |   A
|-------------- | ------
| Documentation | yes
| Bugfix        | yes/no
| BC Break      | yes/no
| New Feature   | yes/no
| RFC           | yes/no
| QA            | yes/no

### Description

Seems to me the workflows in examples are exclusive, they are alternatives and should pick only one.

### CODE REVIEW

**Comment by glensc** [comment]
If the change is valid, maybe it's easier to put the different flows to different sub-directories:

```
➔ tree examples/
examples/
├── release-on-milestone-closed
│   └── release-on-milestone-closed.yml
└── release-on-milestone-closed-triggering-release-event
    └── release-on-milestone-closed-triggering-release-event.yml

2 directories, 2 files
```

**Review by Ocramius — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -13,8 +13,12 @@ workflows into your own project:
 cd /tmp
 git clone https://github.com/laminas/automatic-releases.git
 cd /path/to/your/project
-cp -r /tmp/automatic-releases/examples/.github ./.github
-git add .github
+install -d .github/workflows
```
Comment (Ocramius) [inline README.md]: I'd not change this: `install` is quite obscure, while `cp` is widely understood.

Comment (glensc) [reply README.md]: Changed to `mkdir -p`, Good thing is that you don't have to type commands from here :)

**Review by Ocramius — APPROVED** [review]

**Comment by Ocramius** [comment]
Thanks @glensc

**Pull request merged**